driver/ixsystems/freenasapi.py

```diff
@@ -203,7 +198,7 @@ def invoke_command(self, command_d, request_d, param_list):
             raise FreeNASApiError("Invalid FREENAS command")
         request.get_method = lambda: method
         try:
-            response_d = urllib.request.urlopen(request)
+            response_d = urllib.request.urlopen(request, context=ssl.SSLContext())
             response = self._parse_result(command_d, response_d)
```

Comment:
Add SSLContext enables https authentication. This does not affect http authentication.